app/pods/publications/index/controller.js

```diff
@@ -5,6 +5,7 @@ import { tracked } from '@glimmer/tracking';
 import CONFIG from 'frontend-kaleidos/utils/config';
 import tableColumns from 'frontend-kaleidos/config/publications/overview-table-columns';
 import PublicationFilter from 'frontend-kaleidos/utils/publication-filter';
+import moment from 'moment';
```

Comment:
onnodige import? Linter faalt er op in jenkins builds

![image](https://user-images.githubusercontent.com/22245223/115360096-970d7880-a1bf-11eb-9d6c-cf7d356766cf.png)